wgpu-types/src/lib.rs

```diff
@@ -901,13 +901,15 @@ bitflags::bitflags! {
         /// This mainly applies to a Vulkan driver's compliance version. If the major compliance version
         /// is `0`, then the driver is ignored. This flag allows that driver to be enabled for testing.
         const ALLOW_UNDERLYING_NONCOMPLIANT_ADAPTER = 1 << 3;
-        /// Enable GPU-based validation. Currently, this only changes behavior on the DX12
-        /// backend.
+        /// Enable GPU-based validation. Implies [`Self::VALIDATION`]. Currently, this only changes
```

Comment:
@cwfitzgerald: FYI, added this `Implies …` statement to docs with a4e534979. I think this is correct (why have `GPU_BASED_VALIDATION` with `VALIDATION` off?), just wanted to call this out in case I'm misunderstanding something.